apps/gruenbuch/templates/gruenbuch_page.html

```diff
@@ -0,0 +1,62 @@
+{% extends "base.html" %}
+
+{% load wagtailcore_tags %}
+
+{% block title %}
+  <h1>{{ book.title }}</h1>
+{% endblock title %}
+
+{% block content %}
+  <div class="container">
+    <div class="row">
+      <nav class="col-sm-4 col-md-3 col-xl-2">
+        {% for chapter in chapters %}
+          <div class="m-1 fs-6 fw-bolder">
+            {% if chapter.url == active_chapter.url %}
+              <div class="chapter-link--active">
+                <i class="fas fa-arrow-right"></i>
+                <a href="{{ chapter.url }}">
+                  {{ chapter.page_title }}
+                </a>
+              </div>
+            {% else %}
+              <div class="chapter-link">
+                <a href="{{ chapter.url }}">
+                  {{ chapter.page_title }}
+                </a>
+              </div>
+            {% endif %}
+          </div>
+          <br />
+        {% endfor %}
+      </nav>
+      <div class="col">
+        <h1>
+          {{ active_chapter.page_title }}
+        </h1>
+        <h3>
```

Comment:
Shouldn't this be h2 for a11y? And the we can still style it however we like. (But not sure about the a11y here.)